pandas/tests/indexes/test_base.py

```diff
@@ -48,7 +48,8 @@ def setUp(self):
                             catIndex=tm.makeCategoricalIndex(100),
                             empty=Index([]),
                             tuples=MultiIndex.from_tuples(lzip(
-                                ['foo', 'bar', 'baz'], [1, 2, 3])))
+                                ['foo', 'bar', 'baz'], [1, 2, 3])),
+                            mixedIndex=Index([0, 'a', 1, 'b', 2, 'c']))
```

Comment:
Is it the right way to test it on `difference` and `symmetric_difference`? I needed to exclude it from some tests in common.py (as e.g. `test_argsort`). Still, it triggers RuntimeWarning in `Index.union` (maybe exclude it from `test_union_base` as well?)